### PR TITLE
DP-35809 - Updated BQ dataset name

### PR DESCRIPTION
**Pull request by tom-fleming**
**Description:**
Updated BigQuery dataset name. See test steps from original PR: https://github.com/massgov/openmass/pull/2687

**Jira:** (Skip unless you are MA staff)
DP-35809

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>



## Error Error
```
ENOENT: no such file or directory, open 'changelogs/DP-35809.yml'
Error: ENOENT: no such file or directory, open 'changelogs/DP-35809.yml'
    at Object.openSync (node:fs:573:18)
    at readFileSync (node:fs:452:35)
    at dangerfile.js:60:32
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (dangerfile.js:59:38)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at requireFromString (/tmp/code/node_modules/require-from-string/index.js:28:4)
    at /tmp/code/node_modules/danger/distribution/runner/runners/inline.js:161:68
    at step (/tmp/code/node_modules/danger/distribution/runner/runners/inline.js:52:23)
    at Object.next (/tmp/code/node_modules/danger/distribution/runner/runners/inline.js:33:53)
```
### Dangerfile
```
---------------------^
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bed7c79c27a1b490765e090ceb8e94e02529e911
</p>

File: docroot/modules/custom/mass_bigquery/src/BigqueryStorage.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ public function updateRecords(array $ids): bool {
     $time = \Drupal::time()->getRequestTime();
```
Comment (dstorozhuk) [inline docroot/modules/custom/mass_bigquery/src/BigqueryStorage.php:127]: @tom-fleming  looks good to me, hoverer, I noticed that `$time` is not used in this method. 
probably a good time to remove it?

Comment (tom-fleming) [reply docroot/modules/custom/mass_bigquery/src/BigqueryStorage.php:127]: Thanks @dstorozhuk. This specific `$time` variable is still used, but I removed the unused one on line 167

Comment (dstorozhuk) [reply docroot/modules/custom/mass_bigquery/src/BigqueryStorage.php:127]: oh, ok. I checked the file in PHPStorm, so yes, you are right, it is in another method.

**Pull request merged**